deployments/sepolia/WakuRlnRegistry_Proxy.json

```diff
@@ -1,5 +1,5 @@
 {
-  "address": "0xF471d71E9b1455bBF4b85d475afb9BB0954A29c4",
+  "address": "0xABd93f37833107eDbAb58633ad5463aDEa1F95F4",
```

Comment:
imho the point of having a proxy is that if we deploy a new contract, the proxy address does not change (nwaku nodes can point to the same address, with no changes) but the underlying implementation changes (points to new contract).

not sure its the case with this?

our rlnv1 proxy is 0xF471d71E9b1455bBF4b85d475afb9BB0954A29c4. can we keep that in future deployments and you change the contract that it points to? rlnv1, v2, or any other mod?

Reply:
I needed to deploy a separate instance to test against without breaking the existing deployment

Reply:
A simple `skip: true` is enough for it to use the old deployment